virttest/qemu_devices/qdevice_format.py

```
@@ -310,11 +321,18 @@ def _object_json_format(self, params):
         params = params.copy()
         if "backend" in params:
             params["qom-type"] = params.pop("backend")
+            self._update_args(dev_type, params)
+        elif "qom-type" in params and params["qom-type"]:
```

Ruff (0.14.1)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)